spi.java.hints/apichanges.xml

```diff
@@ -25,6 +25,20 @@
         <apidef name="JavaHintsSPI">Java Hints SPI</apidef>
     </apidefs>
     <changes>
+        <change id="JavaFixUtilities.isPrimary">
+            <api name="JavaHintsSPI"/>
+            <summary>Added JavaFixUtilities.isPrimary() utility</summary>
+            <version major="1" minor="29"/>
+            <date day="5" month="12" year="2017"/>
+            <compatibility addition="yes"/>
+            <description>
+                <p>
+                    The utility method JavaFixUtilities.isPrimary() was added.
+                    This API checks whether a specified tree can be used in
+                    places where a Primary expression is required.
+                </p>
+            </description>
+        </change>
```

Comment:
`project.properties`  already lists version `1.30.0`. You need to bump the version number to a higher value in both `project.properties` and here AND make dependency on that new version from `java.hints` module. 
Otherwise java.hints might autoupdate while this library could remain old (since java.hints requires 1.29, the library without your fix is 1.30 already so it's OK for dependency system). Users will get LinkageError.

Reply:
`apichanges.xml` and `project.properties` updated. I'll take care of the `java.hints` module dependency part in PR #314.